.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Python"]
 	path = Python
 	url = https://github.com/creditornot/blurhash-python.git
+[submodule "Dart"]
```

Comment:
Hey! We prefer to only list repos that we maintain as submodules. A mention in the readme will do for blurhash_ffi 🙂